src/core/algorithms/cfd/cfd_verifier/cfd_verifier.h

```diff
@@ -0,0 +1,74 @@
+#pragma once
+
+#include <memory>
+#include <string>
+#include <utility>
+#include <vector>
+
+#include "algorithms/algorithm.h"
+#include "cfd_stats_calculator.h"
+#include "config/tabular_data/input_table_type.h"
+
+namespace algos::cfd_verifier {
+
+class CFDVerifier : public Algorithm {
+private:
+    config::InputTable input_table_;
+
+    std::vector<std::pair<std::string, std::string>> string_rule_left_;
+    std::pair<std::string, std::string> string_rule_right_;
```

Comment:
I suggest to add an alias for `std::pair<std::string, std::string>` here, it will also make `cfd_verifier.cpp` better